samples/defer3.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func f1() {
+	for i := 0; i < 3; i++ {
```

Comment:
Can we add a one line description for `f1` and `f2`, to illustrate why they print in different order

Reply:
Cool okay. I'll do that.